plugins/base/src/main/java/edu/wpi/first/shuffleboard/plugin/base/widget/GraphWidget.java

```diff
@@ -62,24 +59,23 @@
   @FXML
   private Pane root;
   @FXML
-  private XYChart<Number, Number> chart;
+  private XYChart chart;
+  @FXML
+  private DefaultNumericAxis xAxis;
   @FXML
-  private NumberAxis xAxis;
+  private DefaultNumericAxis yAxis;
   @FXML
-  private NumberAxis yAxis;
+  private ToggleButton autoScrollToggle;
```

Comment:
This looks out of place on the widget. Consider making this an `Action` in the context menu instead, or a `Setting` in the widget properties